README.md

```diff
@@ -6,16 +6,34 @@
 
 Support for Spine-based Java apps running under Google App Engine.
 
-To start a localhost emulator and run tests run `./gradlew check`.
-To start an emulator without running tests `./gradlew startDatastore`.
-To stop datastore use standard system means (e.g. `kill -9 \$(lsof -i:8080)`).
-To run the task successfully, you must have `gcloud` tool properly
-installed and configured: 
+
+##### Testing
+
+###### Preconditions
+
+To run the task successfully, you must have `gcloud` tool properly installed and configured: 
  - install gcloud of the last version
  - login under a Google account when initializing the `gcloud`
- - to run tests you should either select `spine-dev` Google Cloud Console project or create one if it's not accessible for you 
- - if the Google project has different name, change it in `LocalDatastoreStorageFactory`
- 
-The launched emulator will run in `localhost:8080` and will not have any persistence.
-The datastore is cleaned up after each test.
+ - to run tests you should select `spine-dev` Google Cloud Console project
+ - skip Google App Engine setup if not required
+
+
+###### Unix-like
+To start a localhost emulator and run tests run `./gradlew check`.
+To start an emulator without running tests `./gradlew startDatastore`.
+To stop datastore use standard system means (e.g. `kill -9 $(lsof -i:8080)`).
```

Comment:
`To stop the Datastore emulator, just terminate the emulator process. (e.g. ...).